index.html

```diff
@@ -182,17 +182,18 @@ <h4 class="ab-titles skills"> Skills</h4>
       <div class="infography-contact">
         <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
       </div>
-      <div class="form flex-column">
-        <form class="flex-column " action="https://formspree.io/f/mayzgdra" method="POST" class="form-data">
```

Comment:
- [ ] Good job in implementing the following changes, :clap: Your Email Validation works successfully, However, Your form is not submitting the form using formspree, Please ensure that your form submit successfully when it passes the validation checks.